app/factories/flood_risk_engine/steps/form_object_factory.rb

```diff
@@ -38,11 +38,16 @@ def form_object_class_map
         private
 
         def setup_form_object(step)
-          "FloodRiskEngine::Steps::#{step.to_s.classify}Form".constantize
-        rescue
-          Rails.logger.debug(" No Form Object found for step [#{step}] - falling back to object_class map")
-          nil
+          form_name = "FloodRiskEngine::Steps::#{step.to_s.classify}Form"
```

Comment:
this does not seem to be the case, constantize starts to fail if I switch to "#{step.to_s.classify}Form"